test/index.test.js

```diff
@@ -42,7 +42,8 @@ for (const unitTest of fs.readdirSync(`${__dirname}/unit`)) {
       },
       externals: {
         'piscina': 'piscina',
-        'externaltest': 'externalmapped'
+        'externaltest': 'externalmapped',
+        'regexexternal': /\w+-regex/,
```

Comment:
I think it would make more sense to reverse this around so that the key is always the match and the value is always the replacement. Using the value as the replacement in the regex case seems odd otherwise.

So:

```js
  '/\w+-regex/': 'regexexternal-$1': 
```

It might even be nice to permit match replacements on the RHS as well.

Reply:
I went ahead and added this switch, but didn't add replacements for now.